src/RFC-0173_Versioning.md

```diff
@@ -74,7 +74,7 @@ In the Bitcoin P2P protocol messages are preceded by 4
 delimit when a new message starts in a byte stream and also are used to indicate which of the Bitcoin networks the node 
 is speaking on, such as TestNet or Mainnet.
 
-Tari message packets are encapsulated using the Noise protocol so we do not need the delimiting functionality of these 
+Tari message packets are encapsulated using the Noise protocol, so we do not need the delimiting functionality of these 
```

Comment:
The wiremode description is accurate however I wouldn't say that the reason we don't use a message delimiter is because of the noise socket. Once you have a noise socket you are able to send/receive bytes as with any other socket, but those bytes are encrypted over the wire. Using that socket we send [yamux packets](https://github.com/hashicorp/yamux/blob/master/spec.md) and messaging/rpc messages are [length-delimited](https://docs.rs/tokio-util/latest/tokio_util/codec/length_delimited/).

Reply:
Maybe we should change the tense to present?

Reply:
Thanks. Updated